### PR TITLE
Update run.py

### PR DESCRIPTION
**Pull request by YasinEnigma**
Update run.py to solve #756 issue.

### CODE REVIEW

**Comment by rht** [comment]
There is Travis error due to linting: trailing whitespace: https://travis-ci.org/projectmesa/mesa/jobs/620792741#L312.

**Comment by YasinEnigma** [comment]
> There is Travis error due to linting: trailing whitespace: https://travis-ci.org/projectmesa/mesa/jobs/620792741#L312.

I don't know what the problem is that doesn't confirm :(

File: examples/shape_example/run.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
-from shape_example.server import launch_shape_model
+from shape_example.server import server
```
Comment (rht) [inline examples/shape_example/run.py]: There is a trailing whitespace here.

**Comment by Corvince** [comment]
Thanks!

File: examples/shape_example/run.py (hunk 1 above)
Comment (YasinEnigma) [reply examples/shape_example/run.py]: Oh, Thanks.

**Pull request merged**